src/engine/Engine.class.js

```diff
@@ -97,7 +97,7 @@ class Engine {
     this.addValuesMessages = 0
     this.addValuesCount = 0
     this.addFileCount = 0
-    this.forwardedAliveSignalMessages = 0
+    this.forwardedHealthSignalMessages = 0
```

Comment:
this attributes should be managed by the object HealthSignal.

Reply:
This field is not used anyway. I remove it for now and I will implement it with the forward feature that I will do in another PR

Reply:
Will be done with issue #1325